tests/detect-vlan-id/README.md

```diff
@@ -0,0 +1,3 @@
+Test for checking the working of vlan.id keyword by creating rules and matching a crafted packet against them. The packet is an ICMP packet with 3 different VLAN ids [200,300,400].
+
+PCAP created with scapy.
```

Comment:
You can include the scapy script in the PR

Reply:
When you do so, please also include the scapy version used, as things sometimes change from version to version